### PR TITLE
Append Coda Exports to an Existing Manually Coded File

### PR DESCRIPTION
**Pull request by as2388**

### CODE REVIEW

File: tests/traced_data/resources/coda_export_for_append_multiple_schemes.csv [hunk 1]
```diff
@@ -0,0 +1,9 @@
+id;timestamp;owner;data;schemeId;schemeName;deco_codeValue;deco_codeId;deco_confidence;deco_codingMode;deco_timestamp;deco_author
+0;;0;female;3;Gender;F;1-0;0.95;external;Wed Aug 01 2018 12:03:18 GMT+0100 (British Summer Time);
+0;;0;female;59;Age;;;;;;
+1;;1;m;3;Gender;M;1-1;0.95;manual;Wed Aug 01 2018 12:03:18 GMT+0100 (British Summer Time);
+1;;1;m;59;Age;;;;;;
+2;;2;WoMaN;3;Gender;F;1-0;0.95;external;Wed Aug 01 2018 12:03:18 GMT+0100 (British Summer Time);
+2;;2;WoMaN;59;Age;;;;;;
+3;;3;27;3;Gender;;;;;;
+3;;3;27;59;Age;25-35;59-46;0.95;manual;Wed Aug 01 2018 12:03:35 GMT+0100 (British Summer Time);
```
Comment (lukechurch) [inline tests/traced_data/resources/coda_export_for_append_multiple_schemes.csv]: /n at EOF?

Comment (as2388) [reply tests/traced_data/resources/coda_export_for_append_multiple_schemes.csv]: This is what comes out of Coda.

I made an issue for this: https://github.com/AfricasVoices/coda/issues/149

File: core_data_modules/traced_data/io.py [hunk 2]
```diff
@@ -187,7 +191,44 @@ def export_traced_data_iterable_to_coda_with_scheme(data, key_of_raw, key_of_cod
...
+        if prev_f is not None:
+            # Read the previously coded Coda file
+            prev_rows = list(csv.DictReader(prev_f, delimiter=";"))
+            
+            # Exclude items in unique_data which are in the previously coded file.
+            prev_data = set(map(lambda row: row["data"], prev_rows))
+            unique_data = list(filter(lambda td: td[key_of_raw] not in prev_data, unique_data))
+
+            # Detect scheme_id from the previously coded file, and ensure that the previous file had only one scheme id.
+            # TODO: Update this function to support importing multiple coding schemes.
```
Comment (lukechurch) [inline core_data_modules/traced_data/io.py]: Is there an issue for this? If not, let's create one and cross-link it to here

Comment (as2388) [reply core_data_modules/traced_data/io.py]: There is a fix :)

https://github.com/AfricasVoices/CoreDataModules/pull/46

**Pull request merged**